multimap/src/main/java/org/infinispan/multimap/api/embedded/EmbeddedMultimapCacheManagerFactory.java

```diff
@@ -1,4 +1,4 @@
-package org.infinispan.multimap.api;
+package org.infinispan.multimap.api.embedded;
```

Comment:
@tristantarrant If you are ok on the change and so does jenkins, I would like to squash the commits and merge